src/utils/story-helpers.tsx

```diff
@@ -347,3 +351,19 @@ export const OcticonArgType = (iconList: Icon[]) => {
     mapping: icons
   }
 }
+
+export const withSurroundingElements = (
+  Story: React.FC<React.PropsWithChildren<StoryContext>>,
+  context: StoryContext
+) => {
+  const showSurroundingElements =
+    context.globals.showSurroundingElements ?? window.localStorage.getItem('showSurroundingElements') === 'true'
+
+  return (
+    <>
+      {showSurroundingElements ? <a href="https://github.com/primer/react">Primer documentation</a> : ''}
+      {Story(context)}
+      {showSurroundingElements ? <a href="https://github.com/primer/react">Primer documentation</a> : ''}
```

Comment:
Thinking this might improve the clarity of the additional elements

```suggestion
      {showSurroundingElements ? <a href="https://github.com/primer/react">Start link</a> : ''}
      {Story(context)}
      {showSurroundingElements ? <a href="https://github.com/primer/react">End link</a> : ''}
```

Reply:
I am troubled by this because accessibility wise links with the same href should have the same name. Also then the link doesn't actually have a meaningful name. .... but I see the point of your update

Reply:
Do these actually need to navigate to a real link? I'm curious what the benefit of that is. Could it be an empty hash link?